.changeset/ready-seas-ask.md

```diff
@@ -0,0 +1,18 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": deprecation
+---
+
+Deprecation Notice: Events Interfaces Migrated to `@fluidframework/core-interfaces`
+
+The following interfaces and types are now deprecated in `@fluidframework/tree`. It is recommended to import them from either `@fluidframework/core-interfaces` or `fluid-framework`.
+
+- Listeners
+- IsListener
+- Listenable
+- Off
+
+These deprecated interfaces are planned for removal in the Fluid Framework 3.0 release.
```

Comment:
Recommend specifying removal from tree package specifically. They will remain part of FF.